packages/manager/apps/pci-instances/src/pages/instances/create/components/createInstanceForm/CreateInstanceForm.component.tsx

```diff
@@ -48,6 +57,13 @@ export const CreateInstanceForm = () => {
     osType === 'windows' || distributionImageType === 'windows';
 
   const hasMultiMicroRegions = microRegions ? microRegions.length > 1 : false;
+
+  const pricingTypes = selectBillingTypes(deps)(
```

Comment:
could you add a useMemo here to avoid extra computations?
also maybe harmonize `pricingTypes` with `selectBillingTypes`: `pricingTypes` => `billingTypes`

Reply:
`useMemo` ?